config/locales/hyrax.es.yml

```diff
@@ -36,6 +36,8 @@ es:
             description:        "Descripción"
             participants:       "Participantes"
             visibility:         "Lanzamiento y Visibilidad"
+            workflow:           "Flujo de trabajo" # TODO: verify text; added via google translate
```

Comment:
Remove the TODO, this is correct (per https://github.com/projecthydra-labs/hyrax/blob/3efc8eb136f8071ef2f1103415390fb9ef414127/config/locales/hyrax.es.yml#L112)